### ISSUE TITLE
Accessibility issues with screen readers.

### ISSUE THREAD

**Issue by thetechguy-cpu** [jitsi/jitsi-meet#7537]
Hi.
I am a completely blind user of jitsi. I would like to report accessibility issues with jitsi. For reference, I am using NVDA with google chrome. 
### Description:

Here are some of the accessibility issues I found:
- NVDA does not report the toggle state of the button. E.G when clicking on the mute button. 
- Buttons are not marked as buttons to NVDA.
- The more options menu in the participants section is not detectable by NVDA.
- Lack of headings on the page. This would allow a blind person to navigate quickly through the different sections.

### Steps to reproduce:

1.  Launch NVDA.
2.  Go to your jitsi installation.
3.  Try navigating around. 

### Expected behavior:

- Toggleable buttons should report its state to NVDA.
- Buttons should be labeled as buttons.
- Headings on the page for easier navigation.
- The more options menu should be able to be detected by NVDA.

### Actual behavior:

- NVDA does not report the toggle state of togglable buttons.
- The more options button cannot be navigable with NVDA.

### Server information:

- Jitsi Meet version: 4289
- Operating System: Ubuntu 20.04.

### Client information:

- Browser / app version: google chrome 84.0.4147.125
- Operating System: Windows 10, 2004, 64 bit.

**Comment by Echolon** [comment]
Hi, thank you for your report. I labeled it accordingly, there were also other topics on this.

**Comment by thetechguy-cpu** [comment]
Hi.
On the latest version of jitsi (v. 4428) there are accessibility improvements. Now the toggle buttons read its state, however some buttons still don't, such as the raise hand button. Also the more options button for the participants still don't appear on a screen reader. By the way, when arrowing around, I hear an element called SD. What is this? is this the quality of the video? Hopefully these accessibility issues can be addressed in the future :)
